tests/unit/fake_data_root/openstack/var/lib/juju/agents/unit-neutron-openvswitch-1/charm/hooks/charmhelpers/contrib/network/ovs/__init__.py

```diff
@@ -478,7 +478,7 @@ def get_certificate():
         log('Reading ovs certificate from {}'.format(CERT_PATH))
         with open(CERT_PATH, 'r') as cert:
             full_cert = cert.read()
-            begin_marker = "-----SCRUBBED CERTIFICATE-----"
+            begin_marker = end_marker = "-----SCRUBBED CERTIFICATE-----"
```

Comment:
Just and FYI on this and below, it seemed correct but I could not find this file on my install to verify. end_marker is clearly never initialized though...

Reply:
You can you set it to ignore everything under tests/unit/fake_data_root since that is not part of the code and is basically a sosreport created from different types of hosts.

Reply:
Basically any code found under that path is not actually part of hotsos

Reply:
Ack, I'll add an ignore for it, should I just un-do the changes in this directory?

Reply:
Yes please. That data is meant to remain untouched basically.

Reply:
I force pushed, which is probably not the github way, just so used to openstack...

Reply:
I think force pushing is completely fine even on GitHub. Otherwise you would need to squash your commits on merge with the added challenge of producing a squashed commit message.

Reply:
yeah i use force push too so that i always have a single commit